content-repo/extra-docs/releases/20.11.0.md

```diff
@@ -0,0 +1,1015 @@
+Destination Port of the event.
+- **FireEye NX Event ID**
+ID of the event.
+- **FireEye NX Event Rule**
+Rule of the event.
+- **FireEye NX Event Source MAC Address**
+Source MAC Address of the event.
+- **FireEye NX Event Victim IP**
+Source IP of the event.
+- **FireEye NX Event Victim Port**
+Source Port of the event.
+#### Incident Types
+- **FireEye NX Alert**
+
+- **FireEye NX IPS Event**
+
+#### Integrations
+##### FireEye NX  
+FireEye Network Security is an effective cyber threat protection solution that helps organizations minimize the risk of costly breaches by accurately detecting and immediately stopping advanced, targeted, and other evasive attacks hiding in internet traffic.
+#### Layouts
+- **FireEye NX Alert - Summary**
+
+- **FireEye NX IPS Event - Summary**
+
+- **FireEye NX Alert - Mobile**
+
+- **FireEye NX IPS Event - Mobile**
+
+- **FireEye NX Alert - Quick View**
+
+- **FireEye NX IPS Event - Quick View**
+
+
+---
+
+### New: Pulsedive Pack v1.0.0
+#### Integrations
+##### Pulsedive  
+Enrich and analyze any domain, URL, or IP. Pivot to search on data points and linked indicators to investigate risky properties.
+
+
+---
+
+### New: Shift Management - Assign to Next Shift Pack v1.0.0
+#### Playbooks
+##### Assign Active Incidents to Next Shift  
+This playbook reassigns Active Incidents to the current users on call, requires shift management to be setup.   Can be run as a job a few minutes after the scheduled shift change time.
+
+Update the playbook input with a different search query if required.  Will also branch if there are no Incidents that match the query, and no users on call. 
+
+Search results are the default 100 Incidents returned by the query.
+#### Scripts
+##### AssignToNextShift  
+Randomly assigns the incidents to users on call (requires shift management) and users on call.  
+
+https://docs.paloaltonetworks.com/cortex/cortex-xsoar/6-0/cortex-xsoar-admin/users-and-roles/shift-management.html#idf554fd0f-f93b-40cd-9111-1393bf25ac6e
+
+Incident Ids should be passed in as a comma separated list.
+
+
+---
+
+### AWS - ACM Pack v1.0.2
+#### Integrations
+##### AWS - ACM  
+- Fixed an issue where moving the integration to STS failed if the Access Key was previously populated.
+- Upgraded the Docker image to demisto/boto3py3:1.0.0.13191.
+
+---
+
+### AWS - AccessAnalyzer (beta) Pack v1.0.3
+#### Integrations
+##### AWS - AccessAnalyzer (beta)  
+Fixed an issue where moving the integration to STS failed if the Access Key was previously populated.
+
+---
+
+### AWS - Athena (Beta) Pack v1.0.2
+#### Integrations
+##### AWS - Athena (Beta)  
+Fixed an issue where moving the integration to STS failed if the Access Key was previously populated.
+
+---
+
+### AWS - CloudTrail Pack v1.0.3
+#### Integrations
+##### AWS - CloudTrail  
+- Fixed an issue where datetime objects were mishandled in the **aws-cloudtrail-lookup-events** command.
+- Upgraded the Docker image to demisto/boto3:2.0.0.13130.
```

Comment:
remove this one

Reply:
Customers asked to know when a docker has been updated

Reply:
its duplicate...